_data-onboarding/Developer-Guide.md

```diff
@@ -104,32 +139,3 @@ create_device_response = client.post(
 )
 create_device_response.raise_for_status()
 ```
-
-<!-- theme: info -->
```

Comment:
you'll probably want to include this comment before each of the `> ...` block quotes above so they get themed as infoboxes rather than just quotes

Reply:
cool i had no idea what that comment was for lol